bin/validate-commits.js

```diff
@@ -39,29 +39,42 @@ if (argv.installGitHooks) {
   process.exit(0);
 }
 
-const range = argv._[0] || `${getBranch()}..`;
+read(getBranch())
```

Comment:
🍹 Why not put everything in a function and use `async` for everything?

```js
async function run() {
  const commits = await read(getBranch());
  const results = commitValidator.validate(commits);
 
  reporter.printReport(results);

  if (!results.valid && !argv.warning) {
    throw new Error('Commit format error!');
  }
}

run()
  .catch((error) => {
    log(error);
    process.exit(2);
  }); 
```

Reply:
Why not.